repository_service_tuf/cli/admin/ceremony.py

```diff
@@ -426,7 +426,7 @@ def _configure_keys(rolename: str, role: RolesKeysInput) -> None:
         key_count += 1
 
 
-def _check_server(settings):
+def _check_server(settings) -> dict[str, str]:
```

Comment:
Looking into this function I realized that we don't need to have the `else` case in line 445 as if it enters in one of the previous `if` statements the function execution will stop.
If you want you can simplify the function.

Reply:
If it's just a nit, I would leave it like this. I have noticed it as a coding style being done elsewhere as well and it may be considered by some more explicit, easier-to-read, etc.